packages/common/src/chat/chat.tsx

```diff
@@ -438,7 +437,7 @@ export const ChatInternal: FC<ChatProps> = (props: ChatProps) => {
   useEffect(() => {
     if (!pubnub) return;
     /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-    (pubnub as any)._config._addPnsdkSuffix("chat-components", "RCC/__VERSION__");
+    (pubnub as any)._config._addPnsdkSuffix("chat-components", "__PLATFORM__/__VERSION__");
```

Comment:
Please explain it to me

Reply:
Ok so every PubNub SDK attaches `pnsdk` parameter to every call for telemetry purposes (we can pull these out of the logs to gather usage statistics for example). This line uses a hidden JS SDK function to attach some additional info, in this case it's the components' platform and version being used. Those __ strings are replaced with Rollup during build time